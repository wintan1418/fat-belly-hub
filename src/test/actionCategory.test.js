import axios from 'axios';

describe('Api relationship module', () => {
  it(
    '1. verifies the  Categories action method',
    async () => {
      const data = await axios.get('https://www.themealdb.com/api/json/v1/1/categories.php').then(res => res);
      expect(data).toBeInstanceOf(Object);
    },
  );

  it(
    '2. verifies the get one Caetegory info method',
    async () => {
      const data = await axios.get('https://www.themealdb.com/api/json/v1/1/filter.php?c=Seafood').then(res => res);
      expect(data).toBeInstanceOf(Object);
    },
  );
});
