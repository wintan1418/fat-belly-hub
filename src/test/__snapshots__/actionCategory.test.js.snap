// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creates a component returns correctly 1`] = `
<div
  className="categ"
>
  <h3>
    Select a Category 
  </h3>
  <select
    className="cat"
    onChange={[Function]}
    placeholder="Select your category"
  />
</div>
`;
